website/docs/r/network_interface.html.markdown

```diff
@@ -66,6 +66,8 @@ The following arguments are supported:
 
 * `enable_ip_forwarding` - (Optional) Enables IP Forwarding on the NIC. Defaults to `false`.
 
+* `enable_accelerated_networking` - (Optional) Enables Azure Accelerated Networking (AN) using SR-IOV. Only certain VM instance sizes support AN. Refer to [Create VM AN](https://docs.microsoft.com/en-us/azure/virtual-network/create-vm-accelerated-networking-cli). Defaults to `false`.
```

Comment:
> Create VM AN

Could we spell this out as `Create a Virtual Machine with Accelerated Networking` to be clearer? That should allow us to remove the need for the abbreviation above?